src/util.ts

```diff
@@ -358,15 +358,58 @@ export const toCamelCase = (str: string) =>
 export const parseBool = (v: string): boolean | string =>
   ({ true: true, false: false })[v] ?? v
 
-export const parseDotenv = (content: string): NodeJS.ProcessEnv =>
-  content.split(/\r?\n/).reduce<NodeJS.ProcessEnv>((r, line) => {
-    if (line.startsWith('export ')) line = line.slice(7)
-    const i = line.indexOf('=')
-    const k = line.slice(0, i).trim()
-    const v = line.slice(i + 1).trim()
-    if (k && v) r[k] = v
-    return r
-  }, {})
+// prettier-ignore
+export const parseDotenv = (content: string): NodeJS.ProcessEnv => {
+  const kr = /^[a-zA-Z_]+\w*$/
+  const sr = /\s/
+  const e: Record<string, string> = {}
+  let k = ''
+  let b = ''
+  let q = ''
```

Comment:
Why not use more readable variable names?

Reply:
Just a minification challenge without any special reason.

(s)pace(r)egex
(k)ey(r)egex

(e)nv
(k)ey
(b)uffer
(q)oute
(i)gnore
(c)har

Reply:
Lets make it contributor-friendly and leave minification for Vite plugin!